frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -741,9 +750,30 @@ export default {
       }
     },
     async confirm() {
-      await this.$services.example.confirm(this.projectId, this.doc.id)
-      await this.$fetch()
-      this.updateProgress()
+      const canConfirm = this.isAllAffectiveDataAdded()
+      if (canConfirm || this.isProjectAdmin) {
+        this.cannotConfirmAlert = false
```

Comment:
It's better to use positive value rather than negative whenever possible XD so its better to use `canConfirm` and discard  `cannotConfirmAlert` and use `canConfirm` in template, what do you think?

Reply:
or better, you can show the alert only when `canConfirm` is false and user `hasClickedReviewButton` XD

Reply:
noted, thank you for pointing these things out XD I will clean the code following your suggestions 😅

Reply:
I have refactored the code related to this comment :)